packages/cli/lib/publicApps.js

```diff
@@ -0,0 +1,60 @@
+const {
+  fetchPublicAppOptions,
+  selectPublicAppPrompt,
+} = require('./prompts/selectPublicAppPrompt');
+const { EXIT_CODES } = require('./enums/exitCodes');
+const { i18n } = require('./lang');
+const { uiLine } = require('./ui');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const {
+  migrateApp,
+  checkMigrationStatus,
+} = require('@hubspot/local-dev-lib/api/projects');
+
+const i18nKey = 'cli.lib.publicApps';
+
+const fetchPublicApp = async (accountId, accountName, options, migrateApp) => {
+  const { appId } = await selectPublicAppPrompt({
+    accountId,
+    accountName,
+    options,
+    migrateApp,
+  });
+  return appId;
+};
+
+const migratePublicApp = async (accountId, appId, name) => {
+  const response = await migrateApp(accountId, appId, name);
+  return response;
+};
+
+const getMigrationStatus = async (accountId, id) => {
+  const response = await checkMigrationStatus(accountId, id);
+  return response;
+};
+
+const clonePublicApp = async (appId, name, location) => {
+  console.log('Cloning appId', appId);
+  console.log('Name:', name);
+  console.log('Location:', location);
```

Comment:
Agree with what branden said about most of these being kind of superfluous. Remember to switch these to `logger`, wherever you end up putting them

Reply:
Deleted the `clonePublicApp` function in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/62039a6495b1b5eb20d8ddd746508b2daf69a8d4. Going to move these functions to the `migrateApp` folder in a later PR.

Reply:
Removed publicApps util in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/5652758620136accfe8bedcb6f880ea864605757